smaug/operators/control_flow_ops.h

```diff
@@ -55,6 +75,10 @@ class SwitchOp : public Operator {
     }
 };
 
+/** \ingroup Operators
+ * A merge operator takes multiple tensors, all but one of which should be
+ * dead, and copies the one live Tensor to its output.
```

Comment:
I think `all but one of which should be live`.

Reply:
I think we're on the same page logically but maybe it's a grammar mixup? "All but one which should be dead" = "All dead except one", not "All (except one dead tensor)".

I already merged this commit so I'll clarify in a followup.

Reply:
I see :) My bad.. No need to clarify in further commits.